myapp/models.py

```diff
@@ -137,8 +141,14 @@ class Comment(models.Model):
     def __str__(self):
         return f"{self.author}'s comment for '{self.liked_object}': {self.comment[:15]}"
 
+    def save(self, **kwargs):
+        if not self.id:
+            self.date = timezone.now() - timedelta(days=365)
```

Comment:
Ok, what if the year is a leap?) (2016, 2020, 2024, etc?)

Reply:
I changed the way years subtraction
but, if there are no strong requirements, maybe '-timedelta(days=365.25)' can fit as well (in order to be more laconic in code)?